util/index.js

```diff
@@ -0,0 +1,22 @@
+import fetch from 'isomorphic-unfetch'
+
+/**
+ * Make an authenticated request to the backend.
+ */
+export async function secureFetch (url, accessToken, method = 'get', options = {}) {
```

Comment:
This needs to be merged with the secure fetch method in #5.

Reply:
I am also of the opinion that all secure fetches should use the `secureFetch` method introduced by #5.

Reply:
I'm going to create an issue for this. I think we should handle resolving once these PRs are completed.